Dart/src/com/jetbrains/lang/dart/ide/runner/DartConsoleFilter.java

```diff
@@ -32,6 +32,7 @@
   private Collection<VirtualFile> myAllPubspecYamlFiles;
 
   public static final String OBSERVATORY_LISTENING_ON = "Observatory listening on ";
+  public static final String DART_VM_LISTENING_ON = "The Dart VM Service is listening on ";
```

Comment:
Ben say they ended up with [lowercase](https://github.com/flutter/engine/commit/3764a8f0e99e29a91c13c1e230440f239ef96e0d#diff-beebecef6d34b89ff0e076d5ef5eb7c261eb902162a64855e648e14eb013d46bR38) 'service'. For example, see [Flutter engine code](https://github.com/flutter/engine/commit/3764a8f0e99e29a91c13c1e230440f239ef96e0d#diff-beebecef6d34b89ff0e076d5ef5eb7c261eb902162a64855e648e14eb013d46bR38).

Reply:
I'll fix.